src/node/hooks/express/padreadonly.js

```diff
@@ -19,7 +19,7 @@ exports.expressCreateServer = (hookName, args, cb) => {
 
     if (await hasPadAccess(req, res)) {
       // render the html document
-      const html = await exporthtml.getPadHTMLDocument(padId, null);
+      const html = await exporthtml.getPadHTMLDocument(padId, null, null);
```

Comment:
Shouldn't this be:
```suggestion
      const html = await exporthtml.getPadHTMLDocument(padId, null, req.params.id);
```

Reply:
Good catch! It's leaking the writeable id without your change. However, it turns out the whole file seems to be dead code and non-working anyway - `null` as revision parameter is true in this [branch](https://github.com/ether/etherpad-lite/blob/develop/src/node/utils/ExportHtml.js#L32) and `null/100` equals 0 in https://github.com/ether/etherpad-lite/blob/develop/src/node/db/Pad.js#L237 so it's always returning the 0 revision.

`/ro/:id` is no longer used, as we access both writeable and readonly ids via `/p/:id` now. So I'm going to remove the whole thing